cxx/distributions/normal.hh

```diff
@@ -2,19 +2,41 @@
 // See LICENSE.txt
 
 #pragma once
-#include "base.hh"
+#include "globals.hh"
+#include "distributions/base.hh"
```

Comment:
Just base.hh, I think

Reply:
No, all the locations are relative to the cxx directory (because that's what we pass to -I when compiling).